src/app/pages/AccountPage/index.tsx

```diff
@@ -79,15 +79,15 @@ export function AccountPage(props: Props) {
   const selectedNetwork = useSelector(selectSelectedNetwork)
   const { active } = useSelector(selectTransaction)
   const wallets = useSelector(selectWallets)
+  const activeWallet = useSelector(selectActiveWallet)
 
   const balanceDelegations = stake.delegations.reduce((acc, v) => acc + Number(v.amount), 0)
-  const balanceDebondingDelegations = stake.debondingDelegations.reduce((acc, v) => acc + Number(v.amount), 0)
   const balance: BalanceDetails = {
-    available: account.liquid_balance ?? 0,
+    available: parseFloat(activeWallet?.balance.available ?? '') ?? 0,
     delegations: balanceDelegations, //@TODO oasis-explorer : account.debonding_delegations_balance ?? 0,
-    debonding: balanceDebondingDelegations, //@TODO oasis-explorer : account.delegations_balance ?? 0,
-    escrow: account.escrow_balance ?? 0,
-    total: (account.liquid_balance ?? 0) + balanceDelegations + balanceDebondingDelegations,
+    debonding: parseFloat(activeWallet?.balance.debonding ?? '') ?? 0, //@TODO oasis-explorer : account.delegations_balance ?? 0,
+    escrow: parseFloat(activeWallet?.balance.escrow ?? '') ?? 0,
+    total: parseFloat(activeWallet?.balance.total ?? '') ?? 0,
```

Comment:
can you confirm that these have the same meaning? it looks like activeWallet?.balance.xxx are delegations _to_ the account and debonding and total. had the previous code used the delegations _from_ the account?

Reply:
ah the cypress tests are encountering this

Reply:
Yeah, good point. I'll need to check thoroughly what those values are (to or from). Cypress tests are checking the total balance written in bold only and this also seems to be zero for some reason :shrug:

Reply:
In the mean time, https://github.com/oasisprotocol/oasis-wallet-web/pull/408/commits/954a7e905a0e05b449df01e81824c5a4b4de88ef now computes `available` balance from the oasis-node and leave other balances taken from oasis monitor (cypress should pass). I think the `available` balance should always be taken from oasis-node since this balance will be checked, when you to send your tokens.